src/components/EventManagement/EventAttendance/AttendanceList/AttendedEventList.tsx

```
@@ -53,7 +54,13 @@
 
   const event = data?.event ?? null;
 
-  if (loading) return <p>Loading...</p>;
+  if (loading)
+    return (
+      <div data-testid="loading-state" role="status" aria-busy="true">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)